language/Portuguese_Brazil/strings.po

```diff
@@ -7466,92 +7513,70 @@ msgid ""
 "Enter sequence or press %s to cancel\n"
 "%s"
 msgstr ""
+"Enter sequence or press %s to cancel\n"
+"%s"
```

Comment:
This message appears to be untranslated.

Reply:
My bad, thanks for telling me. I wish the translation's file had your comments to help us.